scripts/hudson/narayana.bat

```diff
@@ -0,0 +1,64 @@
+if not defined WORKSPACE (call:fail_build & exit -1)
+
+set NOPAUSE=true
+@echo off
+
+call:comment_on_pull "Started testing this pull request on Windows: %BUILD_URL%"
+
+java -version 2>tmp.txt
+findstr version tmp.txt > version.txt
+FOR /F delims^=^"^ tokens^=2 %%i in (version.txt) DO @set JAVA_VERSION=%%i
+if %JAVA_VERSION% == 9 set MAVEN_OPTS="--add-modules=java.corba,java.xml.bind,java.xml.ws"
+
+IF NOT [%NOTMAIN%] == [] (call:build_spi_pr
+call build.bat clean install %* || (call:comment_on_pull "Tests failed on Windows - Narayana Failed %BUILD_URL%" & exit -1)
+)
+
+rem SHUTDOWN ANY PREVIOUS BUILD REMNANTS
+FOR /F "usebackq tokens=5" %%i in (`"netstat -ano|findstr 9990.*LISTENING"`) DO taskkill /F /PID %%i
+tasklist
+wmic Path win32_process Where "Caption Like '%java.exe%' AND CommandLine Like '%standalone%'" Call Terminate
```

Comment:
Isn't this line killing any appserver instances (Like '%standalone%'). I am reasonably sure that the only such app server instances started by this script were the BlackTie ones. But you have already removed the lines in this script which start instances of the app server. Similarly for the other occurrence (the `fail_build` label).

Reply:
Okay correct, I will remove this